android/src/com/mapswithme/maps/MwmApplication.java

```diff
@@ -107,6 +107,9 @@ public void onCreate()
     initHockeyApp();
     initCrashlytics();
     initPushWoosh();
+    // Workaround to initialize alohalytics for transferring http requests.
+    // Http wrapper should be cut out from the alohalytics in the future.
+    Statistics s = Statistics.INSTANCE;
```

Comment:
OK as a workaround. But need to extract http code into an individual unit and initialize it separately.